plugin/nlspsetting.vim

```diff
@@ -3,18 +3,5 @@ if exists('g:loaded_nlspsettings')
 endif
 let g:loaded_nlspsettings = 1
 
-function! s:complete(arg, line, pos) abort
```

Comment:
What about displaying an error message, as users might be confused if it has been deleted?

Then, after a while, delete it.

```vim
function! s:complete(arg, line, pos) abort
  return join(luaeval('require("nlspsettings.schemas").get_langserver_names()'), "\n")
endfunction

command! -nargs=1 -complete=custom,s:complete NlspConfig echoerr ":NlspConfig has been removed in favor of :LspSettings {server_name}"
command! -nargs=1 -complete=custom,s:complete NlspLocalConfig echoerr ":NlspLocalConfig has been removed in favor of :LspSettings local {server_name}"
command! -nargs=0 NlspBufConfig echoerr ":NlspBufConfig has been removed in favor of :LspSettings buffer"
command! -nargs=0 NlspLocalBufConfig echoerr ":NlspLocalBufConfig has been removed in favor of :LspSettings buffer local"
command! -nargs=1 -complete=custom,s:complete NlspUpdateSettings echoerr ":NlspUpdateSettings has been removed in favor of :LspSettings update {server_name}"
```

Reply:
Done